helm/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rds-chart
-description: A Helm chart for the ACK service controller for Amazon Relational Database Service (RDS)
+description: A Helm chart for the ACK service controller for rds
```

Comment:
You seem to be using a slightly older version of the `code-generator`? I'm assuming that whoever built the version before this commit was using a commit past the `v0.13.0` tag (but before `v0.13.1`). If you pulled the `v0.13.0` tag directly, this change would not be included in that.

For the sake of consistency between the previous commit and this one, would you mind checking out (and regenerating with) this commit: a74c48561a8e6722888cb0d646ae1bb58868281c

That commit is the latest commit after `v0.13.0` before `v0.13.1`.

Reply:
Updated code-generator to `a74c48561a8e6722888cb0d646ae1bb58868281c` and regenerated things. New commit in `4e5eca6e6330a8bd24d8eab164c7002c776c6943`.

Reply:
Much better! Way fewer diffs now. Thanks @rbranche